droplet_deployer/setup_network.js

```diff
@@ -47,6 +47,30 @@ exports = module.exports = function(args) {
 
   var PROVIDER_DETAILS = config.providerDetails[config.provider];
 
+  /**
+   * A helper function to construct SSH options given target machine IP and
+   * config.
+   *
+   * @param {string} ip
+   * @param {object} config - droplet deployer config that also includes SSH
+   *    specific options, e.g. for authentication.
+   */
+  var makeSSHOptions = function (ip, config) {
```

Comment:
No need space between `function` keyword and parenthesis `(` as per our current linter config. `grunt test` should run the lint check. Hope it catches and reports it.